apis/Google.Cloud.Logging.NLog/Google.Cloud.Logging.NLog.Snippets/GoogleStackdriverTargetSnippets.cs

```diff
@@ -40,56 +40,69 @@ public void Overview()
         {
             string projectId = _fixture.ProjectId;
             string logId = $"{_fixture.LogId}-{Guid.NewGuid()}";
+            string fileName = "nlog.xml";
             string resourceName = $"{typeof(GoogleStackdriverTargetSnippets).Namespace}.nlog-template.xml";
 
             // Load the NLog XML configuration, and replace the projectId and logId
-            string xmlString;
+            string xml;
             using (var stream = typeof(GoogleStackdriverTargetSnippets).GetTypeInfo().Assembly
                 .GetManifestResourceStream(resourceName))
             {
                 using (var reader = new StreamReader(stream))
                 {
-                    xmlString = reader.ReadToEnd();
+                    xml = reader.ReadToEnd();
                 }
             }
-            xmlString = xmlString.Replace("PROJECT_ID", projectId).Replace("LOG_ID", logId);
-            var xmlReader = XmlReader.Create(new MemoryStream(Encoding.UTF8.GetBytes(xmlString)));
+            xml = xml.Replace("PROJECT_ID", projectId).Replace("LOG_ID", logId);
+            Assert.False(File.Exists(fileName), "Test would overwrite existing file");
+            //var xmlReader = XmlReader.Create(new MemoryStream(Encoding.UTF8.GetBytes(xmlString)));
```

Comment:
Remove the commented out line?

Reply:
Oops, yes, trying to do things too fast!